AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -95,20 +131,20 @@ private extension SessionHeaderView {
             }
         }.alert(isPresented: $showingFinishAlert) {
             Alert(title: Text(Strings.SessionHeaderView.finishAlertTitle) +
-                Text(session.name ?? Strings.SessionHeaderView.finishAlertTitle_2)
-                +
-                Text(Strings.SessionHeaderView.finishAlertTitle_3),
-                message: Text(Strings.SessionHeaderView.finishAlertMessage_1) +
+                    Text(session.name ?? Strings.SessionHeaderView.finishAlertTitle_2)
```

Comment:
unneeded formatting (as per previous CRs - let's not reformat whole files since we're having some issues with conflicts emerging already)

Reply:
Totally my bad, apologises